main/src/rotate45.cpp

```diff
@@ -16,9 +16,107 @@ QImage Rotate45::processImage(const QImage *workingModel) {
             }
         }
     }
+    blur(rotatedPicture, QPoint(rows, 0), QPoint(cols + rows, cols), QPoint(cols, cols + rows), QPoint(0, rows));
     return rotatedPicture;
 }
 
+static std::pair<double, double> find_coord(QPoint A, QPoint B, QPoint S) {
```

Comment:
```suggestion
static std::pair<double, double> find_coord(QPoint A, QPoint B, QPoint S) {
```
опять snake_case.....  везде его исправь